### PR TITLE
Added Tests for Output Streams

### PR DESCRIPTION
**Pull request by Ferev**

### CODE REVIEW

**Comment by maxhhelman** [comment]
When I run make lint, it seems to work... any idea @timkpaine ?

**Comment by maxhhelman** [comment]
We got all checks to pass :) @timkpaine

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+version: '2'
```
Comment (timkpaine) [inline docker-compose.yml:1]: make a folder called ci and move to folder

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+services:
+  zookeeper:
+    image: wurstmeister/zookeeper:latest
```
Comment (timkpaine) [inline docker-compose.yml:4]: duplicate the images in ci/docker/zookeeper/ e.g. copy the contents of wurstmeister/zookeeper/Dockerfile into ci/docker/zookeeper/Dockerfile so we have a local copy

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    ports:
+      - "2181:2181"
+  kafka:
+    image: wurstmeister/kafka:latest
```
Comment (timkpaine) [inline docker-compose.yml:8]: same as above in ci/docker/kafka/Dockerfile

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      - "9092:9092"
+    environment:
+      KAFKA_LISTENERS: PLAINTEXT://0.0.0.0:9092
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://localhost:9092
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+      KAFKA_CREATE_TOPICS: "tributary:1:1"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sockf
+  echo-server:
+    image: jmalloc/echo-server:latest
```
Comment (timkpaine) [inline docker-compose.yml:19]: same as above in ci/docker/http/Dockerfile

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    ports:
+      - 8080:8080
+  socketio-server:
+    image: voduytuan/docker-socketio:latest
```
Comment (timkpaine) [inline docker-compose.yml:23]: same as above in ci/docker/socketio/Dockerfile

File: server.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+const io = require("socket.io"), server = io.listen(8069);
```
Comment (timkpaine) [inline server.js:1]: make and move to ci/scripts/

**Pull request merged**